cartography/intel/azure/cosmosdb.py

```diff
@@ -5,7 +5,7 @@
 from typing import Generator
 from typing import List
 from typing import Tuple
-
+import ipaddress
```

Comment:
is this an external pypi package? if yes, we need to add it to requirements.txt file

Reply:
no need install this package.